Initial installation fails: Alpine Zigbee2MQTT

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Zigbee2MQTT

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/alpine-zigbee2mqtt.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Alpine

### 📝 Provide a clear and concise description of the issue.

Installation script ends with the following error message:
```
[ERROR] in line 20: exit code 1: while executing command chown -R root:root /etc/zigbee2mqtt /root/.z2m

/dev/fd/63: line 74: SPINNER_PID: unbound variable
```

### 🔄 Steps to reproduce the issue.

Run the installation script with the default values: bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/alpine-zigbee2mqtt.sh)"

### ❌ Paste the full error output (if available).

    ___    __      _               _____   _       __             ___   __  _______  ____________
   /   |  / /___  (_)___  ___     /__  /  (_)___ _/ /_  ___  ___ |__ \ /  |/  / __ \/_  __/_  __/
  / /| | / / __ \/ / __ \/ _ \______/ /  / / __ `/ __ \/ _ \/ _ \__/ // /|_/ / / / / / /   / /   
 / ___ |/ / /_/ / / / / /  __/_____/ /__/ / /_/ / /_/ /  __/  __/ __// /  / / /_/ / / /   / /    
/_/  |_/_/ .___/_/_/ /_/\___/     /____/_/\__, /_.___/\___/\___/____/_/  /_/\___\_\/_/   /_/     
        /_/                              /____/                                                  
  ⚙️  Using Default Settings on node pve03 (🔍  Verbose)
  🖥️  Operating System: alpine
  🌟  Version: 3.21
  📦  Container Type: Privileged
  💾  Disk Size: 0.3 GB
  🧠  CPU Cores: 1
  🛠️  RAM Size: 256 MiB
  🆔  Container ID: 112
  🔍  Verbose Mode: Enabled
  🚀  Creating a Alpine-Zigbee2MQTT LXC using the above default settings
  
  ✔️  Using local for Template Storage.
  ✔️  Using local-lvm for Container Storage.
  ✔️  Updated LXC Template List
  ✔️  LXC Template is ready to use.
  ✔️  LXC Container 112 was successfully created.
  ✔️   Started LXC Container
  ✔️  Set up Container OS
  ✔️  Network Connected: 192.168.0.47
  ✔️  Internet Connected
  ✔️  DNS Resolved github.com to 140.82.121.3
   Updating Container OSUpgrading critical system libraries and apk-tools:
(1/4) Upgrading libcrypto3 (3.3.2-r4 -> 3.5.0-r0)
(2/4) Upgrading libssl3 (3.3.2-r4 -> 3.5.0-r0)
(3/4) Installing libapk2 (2.14.9-r2)
(4/4) Upgrading apk-tools (2.14.6-r2 -> 2.14.9-r2)
Executing busybox-1.37.0-r9.trigger
Continuing the upgrade transaction with new apk-tools:
fetch http://dl-cdn.alpinelinux.org/alpine/latest-stable/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/latest-stable/community/x86_64/APKINDEX.tar.gz
(1/20) Upgrading alpine-baselayout-data (3.6.8-r0 -> 3.7.0-r0)
(2/20) Upgrading musl (1.2.5-r8 -> 1.2.5-r10)
(3/20) Upgrading busybox (1.37.0-r9 -> 1.37.0-r18)
Executing busybox-1.37.0-r18.post-upgrade
(4/20) Upgrading busybox-binsh (1.37.0-r9 -> 1.37.0-r18)
(5/20) Upgrading alpine-baselayout (3.6.8-r0 -> 3.7.0-r0)
Executing alpine-baselayout-3.7.0-r0.pre-upgrade
Executing alpine-baselayout-3.7.0-r0.post-upgrade
(6/20) Installing bridge (1.5-r5)
(7/20) Upgrading ifupdown-ng (0.12.1-r6 -> 0.12.1-r7)
(8/20) Installing openrc-user (0.62.3-r0)
(9/20) Upgrading libcap2 (2.71-r0 -> 2.76-r0)
(10/20) Upgrading openrc (0.55.1-r2 -> 0.62.3-r0)
Executing openrc-0.62.3-r0.post-upgrade
(11/20) Upgrading mdev-conf (4.7-r0 -> 4.8-r0)
(12/20) Upgrading busybox-mdev-openrc (1.37.0-r9 -> 1.37.0-r18)
(13/20) Upgrading alpine-conf (3.19.0-r0 -> 3.20.0-r0)
(14/20) Upgrading alpine-release (3.21.0-r0 -> 3.22.0-r0)
(15/20) Upgrading ca-certificates-bundle (20241010-r0 -> 20241121-r2)
(16/20) Upgrading ssl_client (1.37.0-r9 -> 1.37.0-r18)
(17/20) Upgrading busybox-openrc (1.37.0-r9 -> 1.37.0-r18)
(18/20) Upgrading busybox-suid (1.37.0-r9 -> 1.37.0-r18)
(19/20) Upgrading musl-utils (1.2.5-r8 -> 1.2.5-r10)
(20/20) Upgrading alpine-base (3.21.0-r0 -> 3.22.0-r0)
Executing busybox-1.37.0-r18.trigger
OK: 11 MiB in 36 packages
  ✔️  Updated Container OS
   Installing core dependenciesv3.22.0-133-g618bd9bb163 [http://dl-cdn.alpinelinux.org/alpine/latest-stable/main]
v3.22.0-133-g618bd9bb163 [http://dl-cdn.alpinelinux.org/alpine/latest-stable/community]
OK: 26302 distinct packages available
(1/54) Installing brotli-libs (1.1.0-r2)
(2/54) Installing c-ares (1.34.5-r0)
(3/54) Installing libunistring (1.3-r0)
(4/54) Installing libidn2 (2.3.7-r0)
(5/54) Installing nghttp2-libs (1.65.0-r0)
(6/54) Installing libpsl (0.21.5-r3)
(7/54) Installing zstd-libs (1.5.7-r0)
(8/54) Installing libcurl (8.14.1-r0)
(9/54) Installing curl (8.14.1-r0)
(10/54) Installing libgpg-error (1.55-r0)
(11/54) Installing libassuan (2.5.7-r0)
(12/54) Installing pinentry (1.3.1-r0)
Executing pinentry-1.3.1-r0.post-install
(13/54) Installing libgcrypt (1.10.3-r1)
(14/54) Installing gnupg-gpgconf (2.4.7-r0)
(15/54) Installing gmp (6.3.0-r3)
(16/54) Installing nettle (3.10.1-r0)
(17/54) Installing libffi (3.4.8-r0)
(18/54) Installing libtasn1 (4.20.0-r0)
(19/54) Installing p11-kit (0.25.5-r2)
(20/54) Installing gnutls (3.8.8-r0)
(21/54) Installing libksba (1.6.7-r0)
(22/54) Installing gdbm (1.24-r0)
(23/54) Installing libsasl (2.1.28-r8)
(24/54) Installing libldap (2.6.8-r0)
(25/54) Installing npth (1.8-r0)
(26/54) Installing gnupg-dirmngr (2.4.7-r0)
(27/54) Installing sqlite-libs (3.49.2-r0)
(28/54) Installing gnupg-keyboxd (2.4.7-r0)
(29/54) Installing libbz2 (1.0.8-r6)
(30/54) Installing gpg (2.4.7-r0)
(31/54) Installing libcom_err (1.47.2-r2)
(32/54) Installing e2fsprogs-libs (1.47.2-r2)
(33/54) Installing libintl (0.24.1-r0)
(34/54) Installing libeconf (0.6.3-r0)
(35/54) Installing libblkid (2.41-r9)
(36/54) Installing libmount (2.41-r9)
(37/54) Installing pcre2 (10.43-r1)
(38/54) Installing glib (2.84.2-r0)
(39/54) Installing gpm-libs (1.20.7-r5)
(40/54) Installing slang (2.3.3-r3)
(41/54) Installing libssh2 (1.11.1-r0)
(42/54) Installing mc (4.8.33-r2)
(43/54) Installing nano (8.4-r0)
(44/54) Installing ncurses (6.5_p20250503-r0)
(45/54) Installing newt (0.52.25-r0)
(46/54) Installing openssh-keygen (10.0_p1-r7)
(47/54) Installing libedit (20250104.3.1-r1)
(48/54) Installing openssh-client-common (10.0_p1-r7)
(49/54) Installing openssh-client-default (10.0_p1-r7)
(50/54) Installing openssh-sftp-server (10.0_p1-r7)
(51/54) Installing openssh-server-common (10.0_p1-r7)
(52/54) Installing openssh-server-common-openrc (10.0_p1-r7)
(53/54) Installing openssh-server (10.0_p1-r7)
(54/54) Installing openssh (10.0_p1-r7)
Executing busybox-1.37.0-r18.trigger
Executing glib-2.84.2-r0.trigger
OK: 44 MiB in 90 packages
  ✔️  Core dependencies installed
   Installing Alpine-Zigbee2MQTT(1/11) Installing ca-certificates (20241121-r2)
(2/11) Installing libgcc (14.2.0-r6)
(3/11) Installing libstdc++ (14.2.0-r6)
(4/11) Installing ada-libs (2.9.2-r4)
(5/11) Installing icu-data-en (76.1-r0)
Executing icu-data-en-76.1-r0.post-install
*
* If you need ICU with non-English locales and legacy charset support, install
* package icu-data-full.
*
(6/11) Installing icu-libs (76.1-r0)
(7/11) Installing simdjson (3.12.0-r0)
(8/11) Installing simdutf (7.2.1-r0)
(9/11) Installing nodejs (22.16.0-r2)
(10/11) Installing zigbee2mqtt (2.3.0-r1)
Executing zigbee2mqtt-2.3.0-r1.pre-install
(11/11) Installing zigbee2mqtt-openrc (2.3.0-r1)
Executing busybox-1.37.0-r18.trigger
Executing ca-certificates-20241121-r2.trigger
OK: 139 MiB in 101 packages
chown: /etc/zigbee2mqtt: No such file or directory

[ERROR] in line 20: exit code 1: while executing command chown -R root:root /etc/zigbee2mqtt /root/.z2m

/dev/fd/63: line 74: SPINNER_PID: unbound variable

### 🖼️ Additional context (optional).

_No response_

--- a/install/alpine-zigbee2mqtt-install.sh
+++ b/install/alpine-zigbee2mqtt-install.sh
@@ -14,8 +14,8 @@ network_check
 update_os
 
 msg_info "Installing Alpine-Zigbee2MQTT"
+mkdir -p /root/.z2m /etc/zigbee2mqtt
 $STD apk add zigbee2mqtt
-mkdir -p /root/.z2m
 ln -s /etc/zigbee2mqtt/ /root/.z2m
 chown -R root:root /etc/zigbee2mqtt /root/.z2m
 sed -i -e 's/#datadir="\/var\/lib\/zigbee2mqtt"/datadir="\/etc\/zigbee2mqtt"/' -e 's/#command_user="zigbee2mqtt"/command_user="root"/' /etc/conf.d/zigbee2mqtt

